platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -36,7 +36,7 @@ The following compound operators are supported:
 `NSCompoundPredicateType` | Format string syntax
 --------------------------|---------------------
 `NSAndPredicateType`      | `predicate1 AND predicate2`<br />`predicate1 && predicate2`
-`NSOrPredicateType`       | `predicate1 OR predicate2`<br />`predicate1 || predicate2`
+`NSOrPredicateType`       | `predicate1 OR predicate2`<br /><code>predicate1 &vert;&vert; predicate2</code>
```

Comment:
Great catch!

Reply:
By the way, I think `\|` also works here.

Reply:
Technically this is a bug in jazzy’s markdown parser’s order of operations, as `code || block` should be wrapped first and not interpreted at all.

If you try to use an escape `\` within a code block, I’d expect the behavior to be `code \|\| block` — this is what Github and other Markdown parsers do. Jazzy also behaves this way, apparently except for special Markdown control characters.